### ISSUE TITLE
pdf_book(toc_bib=T) does not work when using biblatex

### ISSUE THREAD

**Issue by qifei9** [rstudio/bookdown#450]
When set 
```
output:
  bookdown::pdf_book:
    citation_package: biblatex
```
the `toc_bib` para does not work.

I guess this is duo to that biblatex uses `\printbibliography` to print bibliography and thus cannot be matched by `r = '^\\\\bibliography\\{.+\\}$'` (`line 189` from `R/latex.R`).

When using biblatex, adding bib to toc could be achieved by manually editing the line `\printbibliography` in the `tex` file to `\printbibliography[heading=bibintoc]`.

**Comment by yihui** [comment]
Are you able to submit a PR to fix it? I'm not really familiar with biblatex. Thanks!

**Comment by qifei9** [comment]
I'm not able to make a PR. My current workaround is add this to preamble: 
```
\let\oldpb\printbibliography
\renewcommand{\printbibliography}{\oldpb[heading=bibintoc]}
```

**Comment by yihui** [comment]
@qifei9 Okay, that should be enough for me to make the change. Thanks a lot!

**Comment by jemus42** [comment]
Should that be working now? Because I have just spent an hour trying to get it to work, including the fix suggested by @qifei9, and now I just gave up and switched back to `natbib`  :/

**Comment by umarcor** [comment]
@jemus42 I could achieve it by adding the following to `preamble.tex`:

```tex
\usepackage{biblatex}
\DeclarePrintbibliographyDefaults{heading=bibintoc}
```

I don't use `toc_bib`, because it does not work; but I use `biblio-title: References`, which works ok together with the fix above. See, for example, https://github.com/LarsAsplund/github-facts/blob/f37b13241c5e8ee6f1a4d2a09573a60b9db73fb0/01-index.Rmd#L12 and https://github.com/LarsAsplund/github-facts/blob/f37b13241c5e8ee6f1a4d2a09573a60b9db73fb0/Rmd/preamble.tex#L4-L5.

**Comment by cderv** [comment]
FWIW if you still need this, I pushed a fix in #1103 that should solve this.
